mmv1/third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -192,6 +192,15 @@ func TestAccContainerCluster_withAddons(t *testing.T) {
 				ImportStateVerify:       true,
 				ImportStateVerifyIgnore: []string{"min_master_version"},
 			},
+			{
+				Config: testAccContainerCluster_withGcpFilestoreCsiDriverEnabled(pid, clusterName),
```

Comment:
Just adding the field in the Addons and updateAddons configs should be fine. We could remove this to save some test time.